scripts/dotnet-cli-build/CompileTargets.cs

```diff
@@ -149,6 +149,11 @@ public static BuildTargetResult RestoreLockedCoreHost(BuildTargetContext c)
             return c.Success();
         }
 
+        [Target(
+            nameof(CompileCoreHost),
+            nameof(BuildCoreHostPackages))]
+        public static BuildTargetResult BuildCoreHost(BuildTargetContext c) => c.Success();
```

Comment:
Yeah.. That could still work. But since you have changed the name of the targets, this change warrants change in the VSO build definitions. 

This need for keeping VSO and our builds scripts in sync is horrible. We must think about it more.